tests/test_view.py

```diff
@@ -124,6 +124,19 @@ def test_view_management(db, bad_db, col, cluster):
     # Test delete missing view with ignore_missing set to True
     assert db.delete_view(view_name, ignore_missing=True) is False
 
+    if db_version >= version.parse("3.11"):
+        res = db.create_view(
+            view_name,
+            view_type,
+            properties={
+                "links": {col.name: {"fields": {}}},
+                "optimizeTopK": [
+                    "BM25(@doc) DESC",
+                ],
+            },
+        )
+        assert "optimizeTopK" in res
```

Comment:
this is failing

Reply:
example CI failure: https://app.circleci.com/pipelines/github/arangodb/python-arango/350/workflows/5c928ec3-5bb4-4371-be55-266fedb562d7/jobs/8415